dask_cuda/initialize.py

```diff
@@ -13,13 +39,16 @@
 
 
 def _create_cuda_context_handler():
-    if int(os.environ.get("DASK_CUDA_TEST_SINGLE_GPU", "0")) != 0:
-        try:
-            numba.cuda.current_context()
-        except numba.cuda.cudadrv.error.CudaSupportError:
-            pass
+    if DASK_USE_ROCM:
+        hiprt.hipCtxGetCurrent()
     else:
-        numba.cuda.current_context()
+        if int(os.environ.get("DASK_CUDA_TEST_SINGLE_GPU", "0")) != 0:
```

Comment:
@younseojava  Why is this check not relevant for the AMD GPU version?

Reply:
Maybe I don't fully understand the purpose of the env variable. Also, with hip port of numba, how can this function be changed? can it be further simplified?